src/plugins/workspace/public/components/workspace_initial/workspace_initial.test.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { act, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { WorkspaceInitial } from './workspace_initial';
+import { coreMock } from '../../../../../core/public/mocks';
+import { createOpenSearchDashboardsReactContext } from '../../../../opensearch_dashboards_react/public';
+
+const mockCoreStart = coreMock.createStart();
+const WorkspaceInitialPage = (props: any) => {
+  const { Provider } = createOpenSearchDashboardsReactContext({
+    ...mockCoreStart,
+    ...{
+      application: {
+        ...mockCoreStart.application,
+        capabilities: {
+          ...mockCoreStart.application.capabilities,
+          dashboards: {
+            isDashboardAdmin: props.isDashboardAdmin,
+          },
+        },
+      },
+    },
+  });
+
+  return (
+    <Provider>
+      <WorkspaceInitial />
+    </Provider>
+  );
+};
+
+describe('WorkspaceInitial', () => {
+  it('render workspace initial page normally when user is dashboard admin', async () => {
+    const { container } = render(WorkspaceInitialPage({ isDashboardAdmin: true }));
```

Comment:
`render(WorkspaceInitialPage({ isDashboardAdmin: true })) ` is a little bit strange to me.
Is `WorkspaceInitialPage` a component or a function?
If it's a function, prefer renaming `WorkspaceInitialPage` to `createWorkspaceInitialPage`.
If it's a component, prefer `render(<WorkspaceInitialPage isDashboardAdmin />)`

Reply:
Sure, I will update the test